notebooks/WFC3/point_spread_function/hst_point_spread_function.ipynb

```diff
@@ -0,0 +1,1551 @@
+{
```

Comment:
<p>Let's group these imports from the same module together:</p>

<pre class="ql-syntax" spellcheck="false">from psf_utilities import ( save_figure, setup_matplotlib, ...
                           ....)
</pre><p> </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/270/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
I agree and I have implemented this change. I have also split the imports onto a new line using the "\" continuation character so that the full list of imported functions is visible without scrolling horizontally in the notebook cell. Please let me know if that is an issue.